### PR TITLE
Integrate compute-service

### PR DESCRIPTION
**Pull request by wilsonianb**
Start contracts at upload
Handle incoming connections in compute service

### CODE REVIEW

File: lib/manager.js [hunk 1]
```diff
@@ -166,9 +81,15 @@ Manager.prototype.pollRunningInstances = function(){
     if (instance.lastCheckedBalance - self.calculateCharge(token) <= self._minBalance) {
       self.chargeToken(token).then(function(newBalance){
         if (newBalance < self._minBalance) {
-          instance.runner.kill();
-          winston.debug('Killed token "' + token + '"');
-          delete instance;
+          request.del('http://127.0.0.1:'+((parseInt(process.env.PORT) || 5000) + 30)+
```
Comment (stevenzeiler) [inline lib/manager.js]: In the future perhaps this behavior could be extracted into some kind of `ComputeServiceClient` library.

Comment (wilsonianb) [inline lib/manager.js]: I'm was planning on replacing this request with:
`events.emit('contract:unfunded', token);`

Comment (stevenzeiler) [inline lib/manager.js]: How about `contact:depleted` ?

Comment (wilsonianb) [inline lib/manager.js]: I like that.

**Comment by stevenzeiler** [comment]
LGTM

**Comment by codius-montecarlo** [comment]
The build is good and I'm seeing 1 +1s from the following folks:

* stevenzeiler

As such, I'll attempt to merge and close this pull request.

**Pull request merged**